migrations/093_questions__partial_credit__add.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE questions ADD COLUMN partial_credit boolean DEFAULT TRUE;
+
+UPDATE questions
+SET partial_credit = (type = 'Freeform');
```

Comment:
Our choice to call the type `Freeform` internally and `v3` in public is an ongoing risk - I'm trusting that `type = 'Freeform'` is what you meant to say here.

Reply:
Agree that the v3/Freeform thing needs to be fixed. I listed it on #853. I believe `Freeform` is correct here because we are inside the DB (`v3` is only used in JSON files).